[Use-case] Add example use-case

supercrazysam opened the pull request:
This is a follow-up from https://discuss.ardupilot.org/t/cmu-nrec-falco-drone-safe-flights-by-agile-collision-avoidance/83300

Adding "Safe flights by agile collision avoidance" to example use-case.

Sincerely,
Sam Shum

Hwurzburg commented:
did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

supercrazysam commented:
> did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

Oh no, for the whole commit, it only consists of text update on the "use-case" page, and also one image upload. That's it.

supercrazysam commented:
> did you mean to include the CSS file? probably need to rebase on master/delete it....also squash the commits to one...

Squashed everything and removed the CSS. The extra CSS was generated by "python update.py --site copter"

Hwurzburg commented on common/source/docs/common-use-cases-and-applications.rst:
```diff
@@ -21,6 +21,17 @@ Use Cases
 .. image:: ../../../images/case-tractor.jpg
 
+
+- `Safe flights by agile collision avoidance <https://discuss.ardupilot.org/t/cmu-nrec-falco-drone-safe-flights-by-agile-collision-avoidance/83300>`__
```
After looking at the built page, this tilte needs working a bit....am going to merge it and then tweak it....thanks

supercrazysam replied:
Hi @Hwurzburg, after talking to the team, I think we can change this the title to "Autonomous drone with agile collision avoidance"

Hwurzburg commented:
thanks...merging

supercrazysam commented:
> thanks...merging

Hi @Hwurzburg, after talking to the team.  

If it is more suitable, I think we can change this the title to "Autonomous drone with agile collision avoidance".

Hwurzburg commented:
Already changed....needs to be shorter but descriptive...I changed it to Collision Avoidance...

supercrazysam commented:
> Already changed....needs to be shorter but descriptive...I changed it to Collision Avoidance...

Hi @Hwurzburg, thank you for letting me know.

I understand. But would it be possible to change it to "autonomous drone with collision avoidance" ?
It is the request from our team. Thank you!
